dds/DCPS/security/AccessControl/LocalAccessCredentialData.cpp

```diff
@@ -6,125 +6,57 @@
 #include "LocalAccessCredentialData.h"
 #include "dds/DCPS/security/CommonUtilities.h"
 
-#include <fstream>
-
 OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 
 namespace OpenDDS {
 namespace Security {
 
 LocalAccessCredentialData::LocalAccessCredentialData()
 {
+
 }
 
 LocalAccessCredentialData::~LocalAccessCredentialData()
 {
+
 }
 
 bool LocalAccessCredentialData::load(const DDS::PropertySeq& props,
                                      DDS::Security::SecurityException& ex)
 {
-  const std::string file("file:");
-  bool permission = false, governance = false, ca = false;
+  std::string name, value;
 
   for (size_t i = 0; i < props.length(); ++i) {
-    const std::string name = props[i].name.in();
-    const std::string value = props[i].value.in();
+    name = props[i].name;
```

Comment:
Change the type of `i` to `unsigned int` to match what sequences use -- this will avoid a truncation warning on some compilers/platforms where size_t is 64-bit.